target/linux/ath79/tiny/base-files/etc/board.d/01_leds

```diff
@@ -96,6 +96,12 @@ tplink,tl-wr941-v4)
 	ucidef_set_led_switch "lan3" "LAN3" "tp-link:green:lan3" "switch0" "0x08"
 	ucidef_set_led_switch "lan4" "LAN4" "tp-link:green:lan4" "switch0" "0x10"
 	;;
+tplink,tl-wr802n-v1)
+	ucidef_set_led_wlan "wlan" "WLAN" "tp-link:blue:system" "phy0tpt"
```

Comment:
Is this vendor behavior? Please use a DTS trigger if that's possible with the diag LEDs ...

Reply:
It's more complicated than that. Manual says that this is supposed to be for WLAN activity, but the actual behaviour I got out of box is different - the light would be solid when Ethernet link is established, and blink otherwise.

So here, I'd rather be consistent with rest of OpenWrt than manufacturer firmware, and drop this trigger altogether, and use this LED as named, for system status.

Reply:
Would be fine for me as well.

Reply:
OK, dropped.